app/src/ActionModal/components/UndelegationModal.vue

```diff
@@ -52,7 +52,7 @@
       </div>
     </TmFormGroup>
     <TmFormGroup
-      v-if="session.addressRole !== `stash`"
+      v-if="!isGlobalUnstake && session.addressRole !== `stash`"
```

Comment:
why not `!sourceValidator`? saves you a whole prop

Reply:
true